core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/Segment2afp.java

```diff
@@ -2493,11 +2493,12 @@ default P getClosestPointTo(Triangle2afp<?, ?, ?, ?, ?, ?> triangle) {
 		final double oy1 = getY1();
 		final double ox2 = getX2();
 		final double oy2 = getY2();
-		for (int i = 0; i < segments.length;) {
-			final double x1 = segments[i++];
-			final double y1 = segments[i++];
-			final double x2 = segments[i++];
-			final double y2 = segments[i++];
+		final int magicFour = 4;
```

Comment:
The number 4 could not be entered in the for loop counter, because of checkstyle complaint of 'magic number'. I thought that is a way of overcoming this problem. Would you have a suggestion?